docker/mesh.config.ts

```diff
@@ -0,0 +1,52 @@
+import { useOpenTelemetry } from '@envelop/opentelemetry';
+import usePrometheus from '@graphql-mesh/plugin-prometheus';
+import { defineConfig as defineServeConfig } from '@graphql-mesh/serve-cli';
+import { trace } from '@opentelemetry/api';
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger';
+import { BasicTracerProvider, SimpleSpanProcessor } from '@opentelemetry/sdk-trace-base';
+
+// Create Jaeger exporter
+const exporter = new JaegerExporter({
+  serviceName: 'my-service-name',
+  host: 'localhost',
+  port: 6832,
+});
```

Comment:
Let's keep Otel and prometheus out of this PR, it needs to be configurable and we are not sure what to include yet.

Reply:
@YassinEldeeb can you please move to another PR? 
Let's try to keep this task simple and efficient

Reply:
sure